.ci/eclipse-compiler-javac.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+ECJ_VERSION="4.6.1"
+
+
+mkdir target/classes
+mkdir target/eclipse
+
+ECJ_EXISTS="$(ls ~/.m2/repository/ | grep ecj-$ECJ_VERSION.jar)"
+
+if ! test -n "$ECJ_EXISTS"; then
+    wget http://central.maven.org/maven2/org/eclipse/jdt/core/compiler/ecj/$ECJ_VERSION/ecj-$ECJ_VERSION.jar -P ~/.m2/repository/
+fi
+
+java -jar ~/.m2/repository/ecj-$ECJ_VERSION.jar -target 1.8 -source 1.8 -cp $1 \
+        -nowarn:[./target/generated-sources/antlr] \
+        -enableJavadoc src/main/java target/generated-sources/antlr \
+    \
+    > target/eclipse/report.txt 2>&1
+
+#check if any fiolations
+RESULT="$(cat target/eclipse/report.txt | grep -E -o '[1-9]+ problem[s}? \([1-9]+ warning[s]?\)')"
+
+#fail build if any fiolations
+if test -n "$RESULT"; then
+    exit 1
```

Comment:
before "exit 1" you need to "cat" report.txt to output to show user all violations.